_data/downloads.yml

```diff
@@ -21,16 +21,33 @@ eol:
 
   - 2.3.8
 
-stable_snapshot:
-
-  url:
-    bz2: https://cache.ruby-lang.org/pub/ruby/stable-snapshot.tar.bz2
-    gz: https://cache.ruby-lang.org/pub/ruby/stable-snapshot.tar.gz
-    zip: https://cache.ruby-lang.org/pub/ruby/stable-snapshot.zip
```

Comment:
Should we have a announcement post because people may depend on these links?

Reply:
Redirect?

Reply:
I do not know how to redirect on Amazon S3.